### DISCUSSION TITLE
Is this feature available in LiveWire?

### DISCUSSION THREAD

**Discussion by Shadecute** [livewire/livewire#3877] [1. Help] [answered]
I just got acquainted with LiveWire. Can the @mention user feature be used on LiveWire?
I searched but did not find any items. Please help me

 for example :https://syropian.github.io/vue-tribute/
and :https://zurb.github.io/tribute/example/

**Comment by joshhanley** [comment, marked as answer]
@Shadecute it's not a built in feature, neither is it built into Vue.

Your examples are an open source package for Vue that can be found here https://github.com/syropian/vue-tribute

So you could create your own package that uses Livewire and Alpine to do something like that yourself, just as the above package has done for Vue.

Hope this helps!
